src/usage/compatibility.md

```diff
@@ -0,0 +1,81 @@
+# Pgvector compatibility
+
+Pgvecto.rs is natively compatible with pgvector at:
+- Create table
```

Comment:
Let's remove it.

Reply:
But if user want to use this mode, there is nowhere to tell him that `Create table` and `Search vectors` is the same other than here.